lib/util.js

```diff
@@ -50,15 +50,16 @@ util = {
      * @param {?Boolean} options.noDefaults - When set to true, id will be retained.
      * @param {?Boolean} options.retainEmptyValues - When set to true, empty property values will be set to ''
      * instead of being deleted.
+     * @param {Boolean} [isV1=false] - When set to true, looks for the pathVariableData property as well.
      * @return {Array} - The set of sanitized entity level variables.
      */
-    handleVars: function (entity, options) {
+    handleVars: function (entity, options, isV1) {
```

Comment:
Don't usually like `Boolean` function parameters. Would be better as an object like `{ isV1: true }` or `{ schema: '1.0' }` etc.

Reply:
I understand. Can we do it anyway? `handleVars(variables, options, true)` does not read very well.

Reply:
Fair enough.